projects/max14906/src/platform/maxim/parameters.h

```diff
@@ -42,6 +42,7 @@
 #include "maxim_irq.h"
 #include "maxim_spi.h"
 #include "maxim_gpio.h"
+#include "maxim_gpio_irq.h"
```

Comment:
is this related to adding stm32 support?

Reply:
yes, because the new basic_example.c after adding support for STM platform requires in the global_desc_param the
GPIO_IRQ_OPS and GPIO_IRQ_EXTRA macros.
GPIO_IRQ_OPS for maxim is max_gpio_irq_ops which is to be found in maxim_gpio_irq.h
GPIO_IRQ_EXTRA in case of maxim should be null.